src/game.js

```diff
@@ -253,18 +271,30 @@
 
                 // update the timer
                 me.timer.update(time);
-
-                // clear the quadtree
-                me.collision.quadTree.clear();
-
-                // insert the world container (children) into the quadtree
-                me.collision.quadTree.insertContainer(api.world);
-
-                // update all objects (and pass the elapsed time since last frame)
-                isDirty = api.world.update(me.timer.getDelta()) || isDirty;
-
-                // update the camera/viewport
-                isDirty = api.viewport.update(me.timer.getDelta()) || isDirty;
+                
+                accumulator += me.timer.getDelta();
+                
+                updateDelta = (me.sys.interpolation) ? me.timer.getDelta() : stepSize;
+    
+                while (accumulator >= stepSize || me.sys.interpolation) {
```

Comment:
I think for the most part that's a game developer problem. When they choose a number for `updatesPerSecond` it should be realistic in terms of how heavy their update logic is.

I guess for this problem, put a max on the accumulator size. Maybe 4x the stepSize.